pxtlib/github.ts

```diff
@@ -492,16 +492,16 @@ namespace pxt.github {
         return repoInfo.fullName + "#" + branchName
     }
 
-    export function listRefsAsync(repopath: string, namespace = "tags", useProxy?: boolean): Promise<string[]> {
-        return listRefsExtAsync(repopath, namespace, useProxy)
+    export function listRefsAsync(repopath: string, namespace = "tags", useProxy?: boolean, noCache?: boolean): Promise<string[]> {
+        return listRefsExtAsync(repopath, namespace, useProxy, noCache)
             .then(res => Object.keys(res.refs))
     }
 
-    export function listRefsExtAsync(repopath: string, namespace = "tags", useProxy?: boolean): Promise<RefsResult> {
+    export function listRefsExtAsync(repopath: string, namespace = "tags", useProxy?: boolean, noCache?: boolean): Promise<RefsResult> {
         const proxy = shouldUseProxy(useProxy);
         let head: string = null
         const fetch = !proxy ?
-            ghGetJsonAsync("https://api.github.com/repos/" + repopath + "/git/refs/" + namespace + "/?per_page=100") :
+            ghGetJsonAsync(`https://api.github.com/repos/${repopath}/git/refs/${namespace}/?per_page=100${noCache ? "&nocache=1" : ""}`) :
```

Comment:
I wonder if `&random=${Math.random()}` would be useful to defeat any browser cache.

Reply:
oops it should be on the call to our cloud... adding browser busting